lib/Backend/Chakra.Backend.vcxproj.filters

```diff
@@ -150,6 +150,7 @@
     <ClCompile Include="$(MSBuildThisFileDirectory)arm64\ARM64UnwindEncoder.cpp">
       <Filter>arm64</Filter>
     </ClCompile>
+    <ClCompile Include="GlobOptArrays.cpp" />

```

Comment:
>GlobOptArrays [](start = 24, length = 13)

Need $(MSBuildThisFileDirectory) here otherwise razzle builds will fail

Reply:
Didn't realize the filters needed that too. thanks for catching.